armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -346,15 +346,7 @@ def _getAverageFuelLFP(self):
         """Compute the average lumped fission products."""
         # TODO: make do actual average of LFPs
         b = self.getCandidateBlocks()[0]
-        lfpCollection = b.getLumpedFissionProductCollection()
-        if lfpCollection:
-            lfpCollectionCopy = lfpCollection.duplicate()
-            fgRemoved = self._getAverageFissionGasRemoved()
-            lfpCollectionCopy.setGasRemovedFrac(fgRemoved)
```

Comment:
`setGasRemovedFrac` was removed because the removal of fission gases was not implemented correctly. In the current ARMI version we were modifying the fission product yields over time and NOT the number densities so this was actually not removing fission gases correctly. There is a much of clean up related to fixing the way the fission gases should actually be removed by modifying block/component number densities directly.